openassessment/xblock/static/js/src/studio/oa_edit.js

```diff
@@ -278,6 +278,66 @@ OpenAssessment.StudioView.prototype = {
     }
 };
 
+/* Get tinyMCE comfig */
+/* jshint unused:false */
+function tinymceCfg() {
```

Comment:
Did you copy this configuration from elsewhere in the codebase? How did you come up with it?

Reply:
I took it from the edx-platform repo:
https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/src/html/edit.coffee#L61